doc/userguide/setting-up-ipsinline-for-windows.rst

```diff
@@ -0,0 +1,47 @@
+Setting up IPS/inline for Windows
+=================================
+
+This guide explains how to work with Suricata in layer 4 inline mode using
+WinDivert on Windows.
+
+First start by compiling Suricata with WinDivert support. For instructions, see
+`Windows Installation
+<https://redmine.openinfosecfoundation.org/attachments/download/1175/SuricataWinInstallationGuide_v1.4.3.pdf>`_.

```

Comment:
@pevma good reminder that we need the windows install docs in this guide, not in a pdf on redmine. Any progress on that end?